### ISSUE TITLE
Decoding unknown objid returns undefined

### ISSUE THREAD

**Issue by Vanuan** [indutny/asn1.js#20]
The following code returns undefined:
```
rfc3280.AttributeType.decode(new Buffer('06092A864886F70D010903','hex'), 'der');
```

As a result, while decoding an object that uses unknown objid, we see a corresponding key missing.

I think, an error should be thrown or objid should be written as a numeric string.

**Comment by felix** [comment]
My current solution to this is here https://github.com/indutny/asn1.js/pull/46

**Comment by indutny** [comment]
Also, I'd rather not use `in`. It should be just fine to check if the value `!== undefined`.

**Comment by felix** [comment]
Seems to be fixed by 7f8a964d33a309896d6d6ef9cdf834ab7f731ad6
